src/CMakeLists.txt

```diff
@@ -83,26 +77,26 @@ if (BENCHMARK_ENABLE_INSTALL)
   install(
```

Comment:
I *think* you simply need to `include(GNUInstallDirs)`
And do s/`BENCHMARK`/`CMAKE`/ here
(and drop those old `BENCHMARK_INSTALL_*` variables)